nightfall-client/src/services/withdraw.mjs

```diff
@@ -111,6 +111,7 @@ async function withdraw(withdrawParams) {
     if (offchain) {
       const peerList = await discoverPeers('Local');
       Object.keys(peerList).forEach(async address => {
+        logger.debug(`offchain transaction - calling ${peerList[address]}/proposer/transfer`);
```

Comment:
The logger message should be '.../proposer/withdraw' instead of '.../proposer/transfer'

Reply:
Good point!